docs/index.rst

```diff
@@ -26,10 +26,10 @@ Web
 The easiest way to get started with tracing is to instrument your web server.
 We support many `Web Frameworks`_. Install the middleware for yours.
 
-Databases
-~~~~~~~~~
+Instrument modules
```

Comment:
Yea, the idea is because we're `patch[ing] widely used Python libraries`, I thought it was not only related to `Databases`. But yea, can rollback the change.